pkgs/tools/misc/phoronix-test-suite/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, php, which, gnused, makeWrapper, gnumake, gcc }:
+
+stdenv.mkDerivation rec {
+  name = "phoronix-test-suite-${version}";
+  version = "8.8.1";
+
+  src = fetchurl {
+    url = "https://phoronix-test-suite.com/releases/${name}.tar.gz";
+    sha256 = "1l5wnj5d652dg02j7iy7n9ab7qrpclmgvyxnh1s6cdnnnspyxznn";
+  };
+
+  buildInputs = [ php ];
+  nativeBuildInputs = [ which gnused makeWrapper ];
+
+  installPhase = ''
+    ./install-sh $out
+    wrapProgram $out/bin/phoronix-test-suite \
+    --set PHP_BIN ${php}/bin/php \
+    --prefix PATH : ${stdenv.lib.makeBinPath [ gnumake gcc ]}
+  '';
+
+  meta = {
```

```suggestion
  meta = with stdenv.lib; {
```
makes the lines below more readable.